### PR TITLE
Update for sprint 9 release

### PR DESCRIPTION
**Pull request by cpaul92**

### CODE REVIEW

File: docs/api-documentation/V2.3/images/inspections-wsd.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+title Inspections\n
+note left of HAOfficer: An inspection can be created while\na work is "In Progess" or "Completed"
+loop foreach inspection evidence file
+opt
+HAOfficer->Works API: POST /files
+Works API->HAOfficer: {file_id}
+end
+end
+HAOfficer->Works API: POST /works/{referenceNumber}/inspections\n(with list of optional file_ids)
+Works API->HAOfficer: {created inspeciton}
```
Comment (RichardEustace) [inline docs/api-documentation/V2.3/images/inspections-wsd.txt:10]: Typo inspeciton?

File: docs/api-documentation/index.html [hunk 2]
```diff
@@ -52,7 +52,7 @@ <h1 class="govuk-heading-m">Releases</h1>
   <li>
     <!-- UPDATE ME to latest/current version -->
     <a class="govuk-link" href="{{ site.baseurl }}/api-documentation/V2.1/">
```
Comment (RichardEustace) [inline docs/api-documentation/index.html:54]:
```suggestion
    <a class="govuk-link" href="{{ site.baseurl }}/api-documentation/V2.2/">
```

File: docs/api-documentation/index.html [hunk 3]
```diff
@@ -75,7 +75,7 @@ <h1 class="govuk-heading-m">Releases</h1>
   <li>
     <!-- UPDATE ME to upcoming latest version -->
     <a class="govuk-link" href="{{ site.baseurl }}/api-documentation/V2.2/">
```
Comment (RichardEustace) [inline docs/api-documentation/index.html:77]:
```suggestion
    <a class="govuk-link" href="{{ site.baseurl }}/api-documentation/V2.3/">
```

Comment (cpaul92) [reply docs/api-documentation/index.html:77]: Good spot.. updated both

**Pull request merged**